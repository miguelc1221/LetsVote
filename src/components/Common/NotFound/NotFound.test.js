import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from './NotFound';

test('NotFound Component should render as expected', () => {
  const component = shallow(<NotFound />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
})
