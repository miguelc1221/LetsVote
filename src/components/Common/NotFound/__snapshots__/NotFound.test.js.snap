exports[`test NotFound Component should render as expected 1`] = `
<div>
  <h1>
    404
  </h1>
</div>
`;
